charts/logging-operator-fluent/templates/fluentbit-cr.yaml

```diff
@@ -16,8 +16,9 @@ spec:
     prometheus.io/port:   "2020"
   image: {{ toYaml .Values.fluentbit.image | nindent 4 }}
   resources: {}
+  tolerations: {{ toYaml .Values.fluentbit.tolerations | nindent 4 }}
```

Comment:
I would ask to put this in an if statement to leave the tolerations off if not defined